### ISSUE TITLE
Jellyfin Media Player client not showing up under sessions

### ISSUE THREAD

**Issue by Hydranet** [CyferShepard/Jellystat#33]
I have one thing I don't understand why that is happening with the sessions view. My AndroidTV, my Android phone and my web-browser from my desktop show up as client under sessions. But when I  use Jellyfin Media Player(Flatpak)  client to play a stream from my desktop as a client it doesn't show up as a session while it does show up in the Jellyfin dashboard as an active stream.

I have no idea what information you would need to debug this but I will just start with this.
Jellyfin Media Player:     1.9.1 
Jellystat: 1.0.4

**Comment by CyferShepard** [comment]
Hi. Can you do me a favor and send me a screenshot of the console log in the web browser. Press F12 and go to Console when your on the home page of Jellystat. Also in regards to the websocket, at the moment it currently does nothing as there was some issues and i had to disable that section temporarily while it gets fixed. This was done to reduce the number of errors in the logs when we know its already broken.

**Comment by CyferShepard** [comment]
Hi @Hydranet , a fix has been pushed, there was a small change made that caused devices without specific device icons to not show in sessions. Please get the latest version and let me know if its fixed

**Comment by Hydranet** [comment]
> Hi. Can you do me a favor and send me a screenshot of the console log in the web browser. Press F12 and go to Console when your on the home page of Jellystat.

Do you want to me to that while I have a stream playing in Jellfyin Media Player from my desktop?

**Comment by CyferShepard** [comment]
Yes please, need to see the logs of when the error is occurring, but i think i replicated it on my side

**Comment by Hydranet** [comment]
I already did a new pull as you suggested in your other reply, let me figure out how to start my container with the old image, then I will create a screenshot.

**Comment by CyferShepard** [comment]
If the new image works then thats fine i wont need the screenshot

**Comment by Hydranet** [comment]
> If the new image works then thats fine i wont need the screenshot

You sure? I'd be more than happy to start an old image if you let me know what the tag was of the old image?

**Comment by CyferShepard** [comment]
No its cool, i was able to replicate the issue on my end almost immediately, was a small oversight that caused the sessions view to break

**Comment by Hydranet** [comment]
Thanks a lot for your help and as state in the feature request I replied to thanks a lot of developing Jellystat!!  AS I am still using Plex with Tautulli currently, it's one of the things I was missing for Jellyfin. Are you going to setup something to allow people to donate to the Jellystat project?

**Comment by CyferShepard** [comment]
I honestly never really thought about that. I just thought that this was something that was needed and since no one was implementing it id do it myself. Il have to give some thought to that but as this is my first major project.

**Comment by Hydranet** [comment]
If I knew how to code I would have helped out but I don't so that's why I thought  donating would be another way to give to the project. I linked your project on a topic I created on Reddit, so hopefully it will get the attention of more people :)

**Comment by CyferShepard** [comment]
Thanks alot, im also hoping more people will pick up on it and help out as im currently the only developer so things are abit slow in terms of development. I just saw your post aswell, thanks for that. Will be closing this issue now since the task is resolved. Maybe also take a look at our Discord if you want to hang out or join the discussion for any new features.
